### ISSUE TITLE
Add the overlays to layman (to show up in gpo.zugaina.org)

### ISSUE THREAD

**Issue by denilsonsa** [sakaki-/rpi3-overlay#35]
http://gpo.zugaina.org/ is a nice website to search for ebuilds across many overlays. It would be nice to have your both overlays ([genpi64](https://github.com/sakaki-/genpi64-overlay) and [sakaki-tools](https://github.com/sakaki-/sakaki-tools), and possibly [others if they are still relevant](https://github.com/sakaki-?tab=repositories&q=overlay)) on that website.

---

<http://gpo.zugaina.org/Overlays> says:

> If you want to add your overlay to this site, you have to add it in layman first. File a bug at [bugs.gentoo.org][1] to do so.

[1]: https://bugs.gentoo.org/enter_bug.cgi?product=Gentoo+Infrastructure&component=Gentoo+Overlays

However, [Gentoo's Bugzilla][1] says:

> Requests related to git.gentoo.org overlay hosting and official repository list maintenance; if you have a GitHub overlay and want to get it add, file a pull request instead: https://wiki.gentoo.org/wiki/Project:Overlays/Overlays_guide

**Comment by necrose99** [comment]
`emerge app-eselect/eselect-repository`


`eselect repository add genpi64 git https://github.com/sakaki-/genpi64-overlay.git` 

`eselect repository add sakaki-tools git https://github.com/sakaki-/sakaki-tools.git` 
done

addable to wiki sure , it would be of use.

**Comment by sakaki-** [comment]
30 Oct 2020: sadly, due legal obligations arising from a recent change in my 'real world' job, I must announce I am **standing down as maintainer of this project with immediate effect**. For the meantime, I will leave the repo up (for historical interest, and since the ebuilds etc. may be of use for others looking to take forward Gentoo on 64-bit RPi systems); however, there will be no further updates to the underlying binhost etc., nor will I be accepting / actioning further pull requests or bug reports from this point. Email requests for support will also have to be politely declined, so, **please treat this as an effective EOL notice**.<br><br>For further details, please see my post [here](https://www.raspberrypi.org/forums/viewtopic.php?p=1750206#p1750206).<br><br>Many thanks for your interest in this project!<br><br>With sincere apologies, sakaki ><
